src/main/java/stream/flarebot/flarebot/commands/general/ShardInfoCommand.java

```diff
@@ -0,0 +1,60 @@
+package stream.flarebot.flarebot.commands.general;
+
+import net.dv8tion.jda.core.JDA;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import org.apache.commons.lang3.text.WordUtils;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShardInfoCommand implements Command {
+    @Override
+    public void onCommand(User sender, TextChannel channel, Message message, String[] args, Member member) {
+        channel.sendMessage(buildTable(FlareBot.getInstance().getClients())).queue();
+    }
+
+    @Override
+    public String getCommand() {
+        return "shardinfo";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Shows info about a shard";
+    }
+
+    @Override
+    public String getUsage() {
+        return "{%}shardsinfo";
+    }
+
+    @Override
+    public CommandType getType() {
+        return CommandType.GENERAL;
+    }
+
+    public String buildTable(JDA[] jdas) {
```

Comment:
If you are gonna make it a method (No reason to) make this private as it isn't used anywhere else.